### PR TITLE
Readme: replace purrr, tidyr and dplyr by poorman

### LINKED ISSUE #218: Make the `README` lighter
We are using tons of packages in `README`, which makes for an unnecessarily resource-intensive GHA to render `README`.

For example, we load the **entire** _tidyverse_ package, which has 108 recursive dependencies, and that is just _one_ of the packages we use.

We should instead be trying to minimize needed dependencies by using our internal packages for data wrangling as much as possible.

Partly related to #167.

### PR DESCRIPTION
**Pull request by etiennebacher**
Close #218

### CODE REVIEW

**Comment by etiennebacher** [comment]
`modelbased` 8.1 is on CRAN but binaries are not available yet. This should be merged in a few days when binaries are available (otherwise the readme rendering will fail)

**Comment by strengejacke** [comment]
Thanks, looks good to me. @IndrajeetPatil ?

**Review by IndrajeetPatil — APPROVED** [review]
LGTM, thanks!

File: README.Rmd [hunk 1]
```diff
@@ -323,7 +324,7 @@ events <- rbind(
 
 # Full join
 data <- full_join(data, events, by = c("date", "Package")) %>%
-  mutate(Package = fct_reorder(Package, count, .fun = sum, .desc = TRUE))
+  mutate(Package = as.character(fct_reorder(Package, count, .fun = sum, .desc = TRUE)))
```
Comment (IndrajeetPatil) [inline README.Rmd:327]: `fct_reorder()` is from _forcats_ package, right?

Comment (etiennebacher) [reply README.Rmd:327]: yes it is. Btw I added `as.character` here because `poorman::full_join` fails otherwise, not sure why

Comment (IndrajeetPatil) [reply README.Rmd:327]: Maybe you can use `data_join()` function here from _datawizard_?

Also, any way to not use `fct_reorder()` and rely only on the base? That'll also help us get rid of _forcats_.

**Comment by IndrajeetPatil** [comment]
As you remove needed deps from README, you can also update the respective GHA:

https://github.com/easystats/easystats/blob/8a5363f2ba1debe7a6896d1bd15143bbc3034eb2/.github/workflows/render-readme.yaml#L38-L52

File: README.Rmd (hunk 1 above)
Comment (strengejacke) [reply README.Rmd:327]: I would say that we should do that later, in a second step. I'm not sure if this is just an one-liner or some more lines of code.

**Comment by etiennebacher** [comment]
@IndrajeetPatil done

**Pull request merged**